lib/connectors/memory.js

```diff
@@ -799,7 +868,9 @@ Memory.prototype.replaceById = function(model, id, data, options, cb) {
     var msg = 'Could not replace. Object with id ' + id + ' does not exist!';
     var error = new Error(msg);
     error.statusCode = error.status = 404;
-    return process.nextTick(function() { cb(error); });
+    return process.nextTick(function() {
```

Comment:
`process.nextTick(cb, err)` while you are at changing this (why is the change needed in the first place?)